tests/rt.conf

```diff
@@ -77,302 +77,314 @@ COMPILE | 32BIT=Y DEBUG=Y
+# COMPILE | REPRO=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp WW3=Y                                                          | standard    | hera.intel     |             |
+# RUN     | fv3_ccpp_gfdlmprad                                                                                                             | standard    | wcoss_cray     |             |
+# RUN     | fv3_ccpp_gfdlmprad                                                                                                             | standard    | wcoss_dell_p3  |             |
+# RUN     | fv3_ccpp_gfdlmprad                                                                                                             | standard    | hera.intel     |             |
+# RUN     | fv3_ccpp_wrtGauss_nemsio_c768                                                                                                  | standard    | hera.intel     |             |
+# RUN     | fv3_ccpp_wrtGauss_nemsio_c768                                                                                                  | standard    | wcoss_cray     |             |
+#
+# # Run one test using the NEMSAppBuilder, to ensure we don't break it:
+# APPBUILD| CCPP_repro                                                                                                                     | standard    | wcoss_cray     |             |
+# APPBUILD| CCPP_repro                                                                                                                     | standard    | wcoss_dell_p3  |             |
+# APPBUILD| CCPP_repro                                                                                                                     | standard    | hera.intel     |             |
+# APPBUILD| CCPP_repro                                                                                                                     | standard    | gaea.intel     |             |
+# APPBUILD| CCPP_repro                                                                                                                     | standard    | jet.intel      |             |
+# RUN     | fv3_ccpp_appbuild                                                                                                              | standard    |                |             |
+#
+# COMPILE | REPRO=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_2017,FV3_GFS_2017_stretched 32BIT=Y                                                     | standard    | wcoss_cray     |             |
+# COMPILE | REPRO=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_2017,FV3_GFS_2017_stretched 32BIT=Y                                                     | standard    | wcoss_dell_p3  |             |
+# COMPILE | REPRO=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_2017,FV3_GFS_2017_stretched 32BIT=Y                                                     | standard    | hera.intel     |             |
+# COMPILE | REPRO=Y CC
```

Comment:
I thought the REPRO and CCPP tests would be turned off with the next commit, not this one. I recall you ran them earlier this week, right? It would be good to confirm that they passed (minus the problems with the GFS_v15 and GFS_v15plus tests, which were my fault) so that when we bring these code changes to dtc/develop we can keep testing CCPP against IPD.

Reply:
Yes, I was running  full test until yesterday morning, when we made final change to IPD physics which changed the results, that's why we temporarily commented out ccpp tests.

https://github.com/ufs-community/ufs-weather-model/pull/36/commits/56407bbcae3516ae13cb58e94b94cb13bfb6ca83

Reply:
Great, thanks for letting me know. We'll do as much as we can to keep up with GFSv16 changes and test them in dtc/develop over the next weeks. No promises, though (depends a lot on UFS release work).